docs/index.html

```diff
@@ -196,29 +197,28 @@
                 vm.route.path.search('Hebrew') !== -1
               ) {
                 for (
-                  var counter = 0,
+                  let counter = 0,
                     elements =
                       document.getElementsByClassName('markdown-section');
                   counter < elements.length;
                   counter++
                 ) {
-                  var item = elements[counter];
+                  const item = elements[counter];
                   item.dir = 'rtl';
                 }
               }
 
               if (
-                vm.route.path.search('Arabic') === -1 &&
-                vm.route.path.search('Hebrew') === -1
+                vm.route.path.search('Arabic' || 'Hebrew') === -1
```

Comment:
I fear that `'Arabic' || 'Hebrew'` will just result in `Arabic`, and we are loosing half of the code

Reply:
Review second commit please

Reply:
Both commits are part of the pr - you need to address concerns about the first commit too

Reply:
do not mark requests for change as resolved without resolving them

Reply:
How can i resolve it?

Reply:
```suggestion
                vm.route.path.search('Arabic') === -1 &&
                vm.route.path.search('Hebrew') === -1
```
To resolve this you can either push a new commit with these changes or click the "Commit suggestion" button